src/components/coin/Coin.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+const Coin = (props) => {
+  const { coin } = props;
+
+  return (
+    <div className="coin">
+      <p className="coin-name">{coin.name}</p>
+      <small>{`$ ${coin.price_usd}`}</small>
+    </div>
+  );
+};
```

Comment:
- [ ] As required for the project to stick to the **[given template design](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/images/metrics_app.png)**, kindly make sure you add the **right arrow** I n each category card

| Template Design | Your Design |
| --------------------- | ------------------- |
| ![Screenshot from 2022-09-10 04-16-25](https://user-images.githubusercontent.com/83042742/189465077-ffa03bfc-fd58-4a73-a2ba-ec519cf2c771.png) | ![Screenshot from 2022-09-10 04-07-45](https://user-images.githubusercontent.com/83042742/189465003-35b43e98-c6cb-44a8-8826-76f9a84030dd.png) |